static/src/stylesheets/base/_faux-block-link.scss

```diff
@@ -54,6 +54,12 @@ a.u-faux-block-link__overlay {
     // this line is needed as all elements have a solid black
     // background in high contrast mode
     opacity: 0;
+
+    // Override the default user agent stylesheet so we don't incur a "Layout"
+    // each time the user taps a link.
+    &:focus {
+        outline: none;
+    }
```

Comment:
For those interested, this is a bug in Blink: https://code.google.com/p/chromium/issues/detail?id=522852